modules/graphite.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
+
+__version__ = 1.0
+__cacheable__ = True
+
+import urllib2
+import json
+import ConfigParser
+from application import config
+
+CONFIG_KEY = 'module_temp'
+
+
+def update_document(data):
+	remote = urllib2.urlopen("http://graphite.at.hskrk.pl/render/?&target=hs.hardroom.*&format=json&maxDataPoints=1&from=-1min", timeout=2)
```

Comment:
Maybe location part of URL should be in config ? Also parameter `target` coulde be to